src/bosh-softlayer-cpi/softlayer/common/agent_options.go

```diff
@@ -42,6 +42,14 @@ func (o AgentOptions) Validate() error {
 	return nil
 }
 
+func (o *AgentOptions) GetVcapPassword() string {
+	return o.vcapPassword
+}
+
+func (o *AgentOptions) SetVcapPassword(vcapPassword string) {
```

Comment:
@mattcui 

But this still makes the changing and getting of the password public? So nothing changed.

I think another way to phrase the question I am trying to ask is whether `VcapPassword` can be private and only used within the package? Or does it have to be public?